src/atoms/Tab/Tab.native.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { View, TouchableOpacity } from 'react-native';
+import styled from 'styled-components/native';
+import Flex from '../Flex';
+import Text from '../Text';
+import { getColor, getIconKeys, makePxValue } from '../../_helpers/theme';
+import Icon from '../Icon';
+import Space from '../Space';
+import automation from '../../_helpers/automation-attributes';
+import isEmpty from '../../_helpers/isEmpty';
+
+const styles = {
+  label: {
+    borderBottomWidth({ isActive }) {
+      if (isActive) {
+        return '1px';
```

Comment:
can we use `makePxValue` helper here?

Reply:
Not sure. We can have discussion on this.